Add additional details around healthcheck operation for TFE (#1068)
hashicorp/terraform-website: content/source/docs/enterprise/admin/monitoring.html.md

Terraform Enterprise provides a `/_health_check` endpoint on the instance. If Terraform Enterprise is up, the health check will return a `200 OK`.

> The `/_health_check` endpoint operates in 2 modes

Reviewer: ```suggestion
The `/_health_check` endpoint operates in 2 modes:
```

- Full check
- Minimal check

With full check, the service will attempt to verify status of internal component  and PostgreSQL, in contrast to minimal check in which only returns `200 OK`.

> In normal circumstance, full check will be performed during start up of the instance, once Terraform Enterprise started up in active running state, any subsequent checks will be performed in minimal check mode.

Reviewer: ```suggestion
In normal circumstances, a full check will be performed during the start of an instance and once Terraform Enterprise is in an active running state, any subsequent checks will perform a minimal check.
```

Reviewer 2: ```suggestion
The endpoint's default behavior is to perform a full check during startup of the instance, and minimal checks after Terraform Enterprise is active and running.
```

> -> **Note:** If you wish to perform full check forcefully, additional query parameter is required with `/_health_check?full=1` and extra caution as every single call will make contact to internal component and PostgreSQL.

Reviewer: ```suggestion
-> **Note:** If you wish to force a full check, an additional query parameter is required: `/_health_check?full=1`. Take extra caution as every call will make contact internal components and PostgreSQL, increasing system load and latency.
```

Reviewer 2: ```suggestion
-> **Note:** If you wish to force a full check, an additional query parameter is required: `/_health_check?full=1`. Take extra caution as every call will make requests to internal components and PostgreSQL, increasing system load and latency.
```
